lib/priority_spidr/priority_agent.rb

```diff
@@ -0,0 +1,47 @@
+require "spidr/agent"
```

Comment:
Actually not sure if we have to require these either?

Reply:
Huh, removed it and it worked. I thought I would have needed it in order to inherit from `Agent`. I'm assuming ruby does that automatically by having scoped it as part of the `Spidr` module?